src/ops/sequences.jl

```diff
@@ -1,244 +1,53 @@
-"""
-Creates a constant `Tensor`.
-"""
-@op function constant(tensor; name=nothing)
-    local desc
-    with_op_name(name, "Const") do
-        desc = NodeDescription("Const")
-        tensor = RawTensor(tensor)
-        desc["dtype"] = eltype(tensor)
-        desc["value"] = tensor
+import .Ops:
+    random_uniform,
+    random_standard_normal,
+    random_shuffle
+
+@op function constant(value; dtype=nothing, kwargs...)
+    if dtype === nothing
+        if isa(value, AbstractString)
+            dtype = String
+        else
+            dtype = eltype(value)
+        end
     end
-    Tensor(Operation(desc), 1)
+    Ops.const_(; value=value, dtype=dtype, kwargs...)
 end
 
 for f in [:zeros, :ones]
-    @eval Base.$f{T}(::Type{Tensor{T}}, shape...) = constant($f(T, shape...))
     @eval Base.$f(::Type{Tensor}, args...) = $f(Tensor{Float32}, args...)
+    @eval Base.$f(::Type{Tensor}, args::Tuple) = $f(Tensor, args...)
+    @eval Base.$f{T}(::Type{Tensor{T}}, args...) = constant($f(T, args...))
+    @eval Base.$f{T}(::Type{Tensor{T}}, args::Tuple) = constant($f(T, args))
 end
 
-"""
-Outputs random values from a uniform distribution.
-
-The generated values follow a uniform distribution in the range `[minval, maxval)`.
-The lower bound `minval` is included in the range, while the upper bound `maxval` is excluded.
-
-For floats, the default range is `[0, 1)`. For ints, at least `maxval` must be specified explicitly.
-
-In the integer case, the random integers are slightly biased unless
-`maxval - minval` is an exact power of two.
-The bias is small for values of `maxval - minval` significantly smaller than the
-range of the output (either 2**32 or 2**64).
-
-Args:
-* `shape`: A one dimensional `Tensor` or array containing the shape of the output `Tensor`.
-* `minval`: A zero dimensional `Tensor` or value of type `dtype`. Lower bound on random values.
-* `maxval`: A zero dimensional `Tensor` or value of type `dtype`. Upper bound on random values.
-* `seed`: An integer to seed the RNG with. Defaults to `0`.
-* `dtype`: Optional datatype of random values generated. Default is `Float32`.
-
-Returns:
-A `Tensor` of the specified `shape` and `dtype` containing random values.
-"""
-@op function random_uniform(shape, minval, maxval; name=nothing, seed=0, dtype=Float32)
```

Comment:
This change broke `nn.dropout`
The new imported op has a different signature:
and does not support minval, maxval or dtype.
`random_uniform(shape; seed=0, seed2=0)`

I'll make a PR shortly to bring this back,